File: src/main/java/no/nav/melosys/eessi/models/sed/nav/SpesielleOmstendigheter.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;

@JsonInclude(Include.NON_NULL)
@Data
public class SpesielleOmstendigheter {

    private String type;

    private String beskrivelseannensituasjon;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/SpesielleOmstendigheter.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class SpesielleOmstendigheter(
    var type: String? = null,
    var beskrivelseannensituasjon: String? = null
)
